skills/rag-blueprint/references/shutdown.md

```
@@ -67,46 +67,39 @@ If K8s deployment was detected, use the release name and namespace from `helm li
-```
+To also clean up persistent data (only if the user requests full cleanup and confirms), delete the leftover `nimcache` and `pvc` resources in the namespace with `kubectl`.
 
 ## Step 3: Verify Everything Stopped
 
 ```bash
 echo "=== REMAINING ===" && docker ps --format "table {{.Names}}\t{{.Status}}" 2>/dev/null; echo "=== K8S ===" && kubectl get pods -n rag 2>/dev/null | head -10 || echo "NOT_K8S"; helm list -n rag 2>/dev/null || true
 ```
 
-If any RAG-related containers remain, force remove:
+If any RAG-related containers remain, list them, confirm with the user, then remove them individually by name:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)